package/PKGBUILD

```diff
@@ -4,7 +4,7 @@ targets=(
 )
 pkgname="carbonio-files-db"
 pkgver="0.1.3"
-pkgrel="1"
+pkgrel="2209261709"
```

Comment:
It is only for internal use. In this way, when a new PR is merged in develop, the package generated is always different from the previous one. Other teams use the `SNAPSHOT` keyword, other ones the timestamp, we decided to use a more human readable format: year-month-day-hours-minutes.

In the release phase it will be changed in `1` or more when an hotfix is made

Reply:
All clear